pipelined/bevy_wgpu2/src/render_resource_context.rs

```diff
@@ -4,8 +4,8 @@ use crate::{
 };
 use bevy_render2::{
     pipeline::{
-        BindGroupDescriptor, BindGroupDescriptorId, BindingShaderStage, PipelineDescriptor,
-        PipelineId,
+        BindGroupDescriptor, BindGroupDescriptorId, BindingShaderStage, ComputePipelineDescriptor,
+        PipelineDescriptor, PipelineId,
```

Comment:
We might want to rename `PipelineDescriptor` to `RenderPipelineDescriptor`?

Reply:
Yeah. That is both more descriptive and in-line with wgpu (which might ultimately replaces this api).

Reply:
Done!